samples/extensions/full_screen_exclusive/README.md

```diff
@@ -0,0 +1,317 @@
+<!--
+- Copyright (c) 2023, Holochip Corporation
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
+# Full Screen Exclusive
+
+## Overview
```

Comment:
Maybe it's because I'm not a native (English) speaker, but this paragraph is kinda hard to read. Can we simplify the sentences here a little bit? From what I understand from this paragraph, the extension is only required to get exclusive full screen on Windows versions prior to 11?